xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -843,20 +843,20 @@ void CEdl::SetLastEditTime(int editTime)
   m_lastEditTime = editTime;
 }
 
-bool CEdl::GetNearestEdit(bool bPlus, const int iSeek, Edit* pEdit) const
+bool CEdl::GetNearestEdit(bool bPlus, int seekTime, Edit* pEdit) const
```

Comment:
@yol mentioned this a while ago. From https://stackoverflow.com/questions/111933/why-shouldnt-i-use-hungarian-notation:

```
Most people use Hungarian notation in a wrong way and are getting wrong results.

In short, Hungarian Notation where you prefix your variable names with their type (string) (Systems Hungarian) is bad because it's useless.

Hungarian Notation as it was intended by its author where you prefix the variable name with its kind (using Joel's example: safe string or unsafe string), so called Apps Hungarian has its uses and is still valuable.
```

I see what you mean. Ok if drop it all together and apply the same to `bPlus` and `pEdit` ?

Reply:
Done the above in the lastest force push

Reply:
My whole point is consistency across all parameters of the method. No hard feeling whether it is Hungarian Notation or not (due to lack of coding guidelines).

Reply:
FWIW, there is no lack of guidelines in this point

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-variables

"Type prefixing (Systems Hungarian notation) is discouraged."

Reply:
Thanks @yol, missed that so far.